appcontainer/SizingModeModel.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2021 Extremely Heavy Industries Inc.
+ */
+import {HoistModel, XH} from '@xh/hoist/core';
+import {action, observable, makeObservable} from '@xh/hoist/mobx';
+import {throwIf} from '@xh/hoist/utils/js';
+
+/**
+ * Manage Sizing Mode.
+ *
+ *  @private
+ */
+export class SizingModeModel extends HoistModel {
+
+    @observable sizingMode = null;
+
+    modes = [
```

Comment:
am fine with this being the general `SizingMode` even if right now it only refers to Grids

Reply:
Agree - I think it's a feature, even in advance of any other Hoist components reading it or changing as a result. 

E.g. in one of our current client projects, we talked about other aspects of very data-driven (but not grid) widget internals respecting sizing mode changes.